sdk/trace/src/main/java/io/opentelemetry/sdk/trace/export/BatchSpanProcessor.java

```diff
@@ -324,7 +324,7 @@ private void exportCurrentBatch() {
         } else {
           logger.log(Level.FINE, "Exporter failed");
         }
-      } catch (RuntimeException e) {
+      } catch (Throwable e) {
```

Comment:
Can you call `ThrowableUtil.propagateIfFatal` here?

https://github.com/open-telemetry/opentelemetry-java/blob/main/sdk/common/src/main/java/io/opentelemetry/sdk/internal/ThrowableUtil.java#L19

Reply:
I think we should not propagate the exception, the worker thread will be killed as usual if call `ThrowableUtil.propagateIfFatal`.

Reply:
Yes for fatal exceptions like OutOfMemory this needs to happen to let the runtime know of the unrecoverable problem. We always propagateIfFatal in this sort of pattern.

Reply:
OK, it sounds reasonable.